### DISCUSSION TITLE
domino/12.0.0/admin/inst_onetouch_invoking_json_withoutdocker

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#99] [Q&A]
# domino/12.0.0/admin/inst_onetouch_invoking_json_withoutdocker

Complete this procedure to invoke one-touch Domino setup with parameters in a JSON file if you run Domino on Windows or UNIX.

https://help.hcl-software.com/domino/12.0.0/admin/inst_onetouch_invoking_json_withoutdocker.html

**Comment by dperarnaud** [comment]
Linux

cd /local/notesdata
su notes
/opt/hcl/domino/bin/server -autoconf /pathtosetup/setup.json
